internal/resolver/dns/dns_resolver_test.go

```diff
@@ -1368,3 +1368,51 @@ func (s) TestMinResolutionInterval_NoExtraDelay(t *testing.T) {
 	case <-stateCh:
 	}
 }
+
+func (s) TestIPv6Address(t *testing.T) {
+	tests := []struct {
+		name              string
+		target            string
+		hostLookupTable   map[string][]string
+		srvLookupTable    map[string][]*net.SRV
+		txtLookupTable    map[string][]string
+		wantAddrs         []resolver.Address
+		wantBalancerAddrs []resolver.Address
+		wantSC            string
+	}{
+		{
+			name:   "ipv6_address",
```

Comment:
where are you checking for ip address mentioned here https://github.com/grpc/grpc-go/issues/3272 ?